tests/playbooks/roles/install-csi-cinder/tasks/main.yaml

```diff
@@ -0,0 +1,215 @@
+
+      sed -i "s#docker.io/k8scloudprovider/cinder-csi-plugin:latest#{{ remote_registry_host }}/cinder-csi-plugin-amd64:{{ github_pr }}#" manifests/cinder-csi-plugin/cinder-csi-nodeplugin.yaml
+
+- name: Deploy openstack-cloud-controller-manager
+  shell:
+    executable: /bin/bash
+    cmd: |
+      set -x
+
+      cat <<EOF | kubectl apply -f -
+      ---
+      apiVersion: v1
+      kind: ServiceAccount
+      metadata:
+        name: cloud-controller-manager
+        namespace: kube-system
+      ---
+      apiVersion: rbac.authorization.k8s.io/v1
+      kind: ClusterRoleBinding
+      metadata:
+        name: system:cloud-controller-manager
+      roleRef:
+        apiGroup: rbac.authorization.k8s.io
+        kind: ClusterRole
+        name: cluster-admin
+      subjects:
+      - kind: ServiceAccount
+        name: cloud-controller-manager
+        namespace: kube-system
+      ---
+      apiVersion: apps/v1
+      kind: Deployment
+      metadata:
+        name: openstack-cloud-controller-manager
+        namespace: kube-system
+        labels:
+          k8s-app: openstack-cloud-controller-manager
+      spec:
+        replicas: 1
+        selector:
+          matchLabels:
+            k8s-app: openstack-cloud-controller-manager
+        template:
+          metadata:
+            labels:
+              k8s-app: openstack-cloud-controller-manager
+          spec:
+            tolerations:
+            - key: node.cloudprovider.kubernetes.io/uninitialized
+              value: "true"
+              effect: NoSchedule
+            - key: node-role.kubernetes.io/master
+              effect: NoSchedule
+            serviceAccountName: cloud-controller-manager
+            imagePullSecrets:
+            - name: openlab
+            containers:
+              - name: openstack-cloud-controller-manager
+                image: docker.io/k8scloudprovider/openstack-cloud-controller-manager:latest
+                args:
+                  - /bin/openstack-cloud-controller-manager
+                  - --v=4
+                  - --cloud-config=/etc/config/cloud.conf
+                  - --cloud-provider=openstack
+                  - --use-service-account-credentials=true
+                  - --bind-address=127.0.0.1
+                volumeMounts:
+                  - mountPath: /etc/config
+                    name: cloud-config-volume
+                    readOnly: true
+            hostNetwork: true
+            volumes:
+            - name: cloud-config-volume
+              secret:
+                secretName: cloud-config
+      EOF
+
+- name: Wait for openstack-cloud-controller-manager up and running
+  shell:
+    executable: /bin/bash
+    cmd: |
+      kubectl -n kube-system get pod | grep openstack-cloud-controller-manager | grep Running
+  register: check_occm
+  until: check_occm.rc == 0
+  retries: 24
+  delay: 5
```

Comment:
adding as seperate comment
@jichenjc Does this return error if we dont have occm pod running? I still see ignore_errors: yes , so clarifying

Reply:
yes, I need remove that line